src/account/builder.rs

```diff
@@ -214,7 +214,9 @@ pub(crate) async fn get_first_public_address(
             account_index,
             0..1,
             false,
-            GenerateAddressMetadata { syncing: true },
+            Some(GenerateAddressOptions {
+                ledger_nano_prompt: true,
+            }),
```

Comment:
Should it be None instead ?

Reply:
Yes, at least currently the behavior is like `None`